storybookjs/storybook
# SyntaxHighlighter: Put formatted code to clipboard

## What I did

I've added some changes to put formatted code to clipboard if `SyntaxHighlighter.format=true`

## How to test

1. Open ?path=/story/basics-syntaxhighlighter--bordered-copy-able in examples/official-storybook
1. Click 'Copy' button
1. Check that your clipboard equals to
```
import { Good, Things } from 'life';

const result = () => <Good><Things /></Good>;

export { result as default };
```
